doc/building.md

```diff
@@ -57,3 +57,27 @@ Openconsole has three configuration types:
 - AuditMode
 
 AuditMode is an experimental mode that enables some additional static analysis from CppCoreCheck.
+
+## Updating Nuget package references
+Updating Nuget package references must be done using the snippet listed below because of the way packages are consumed by the projects.

```

Comment:
```suggestion
Certain Nuget package references in this project, like `Microsoft.UI.Xaml`, must be updated outside of the Visual Studio NuGet package manager. This can be done using the snippet below.
```